Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
> +      @filename = Utils.popen_read("npm pack")

I'd rather not rely on a magic instance variable. Can we have a getter method for this instead?